go/client/cmd_simplefs_write.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+const writeBufSizeDefault = 1600
+
+// CmdSimpleFSWrite is the 'fs write' command.
+type CmdSimpleFSWrite struct {
+	libkb.Contextified
+	path    keybase1.Path
+	flags   keybase1.OpenFlags
+	offset  int64
+	bufSize int
+}
+
+// NewCmdSimpleFSWrite creates a new cli.Command.
+func NewCmdSimpleFSWrite(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "write",
+		ArgumentHelp: "<path>",
+		Usage:        "write input to file",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdSimpleFSWrite{Contextified: libkb.NewContextified(g)}, "write", c)
+		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "a, append",
+				Usage: "add to existing file",
+			},
+			cli.IntFlag{
+				Name:  "b, buffersize",
+				Value: writeBufSizeDefault,
+				Usage: "write buffer size",
+			},
+		},
+	}
+}
+
+// Run runs the command in client/server mode.
+func (c *CmdSimpleFSWrite) Run() error {
+
+	// Disable the default signal/interrupt handler so that
+	// we can uplodad the buffer after ctl-C
+	signal.Reset(os.Interrupt, syscall.SIGTERM, os.Kill)
```

Comment:
I don't think this is a good idea. You still need to exit ASAP if you catch a signal.  To just ignore it doesn't seem right.